wallet/src/Pos/Wallet/Aeson/WalletBackup.hs

```diff
@@ -122,7 +122,7 @@ instance ToJSON IndexedAccountMeta where
 
 instance ToJSON WalletBackup where
     toJSON (WalletBackup skey wMeta wAccounts) = object
-        [ "walletSecretKey" .= B64.encode (Bi.serialize' prvKey)
+        [ "walletSecretKey" .= B64.encode (Bi.serialize' $ SecretKey prvKey)
```

Comment:
Why did you have to add this wrapper here? Did it work incorrectly before? I thought this task was about adding tests, but here you're changing the logic, is this a bugfix?

Reply:
This task is about adding some instances for types with `Bi` instances. Previously `XPrv` had `Bi` instance, but I find it wrong (because it's implementation detail) and asked to delete that instance. I think `XPrv` shouldn't be used directly, the purpose of `SecretKey` newtype is to hide it.